wiki/debugging.md

```diff
@@ -0,0 +1,76 @@
+# Debugging data issues
+
+Many issues can arise in the Metro galaxy, from the shallowest part of the frontend to the deepest end of the backend. Tackling data inaccuracies requires a strategic approach. This guide traces an easeful, step-by-step process for efficient debugging. 
+
+DON'T PANIC. 
+
+## Identifying the point of failure
+
+[point to the charts in the overview that describe the data pipeline]
+
+Data travels along a complicated path to arrive in Metro Councilmatic. A disruption in the data pipeline can happen at four different points. 
+
+1. Legistar - the Legistar API or web interface did not provide accurate data 
```

Comment:
what about a point 0: data entry by metro (or other legislative body) staff?

Reply:
that seems directly linked to Legistar. but I think I can add a clause, like, "likely because Metro staff did not enter accurate data"